Makefile

```diff
@@ -61,4 +63,16 @@ k8s-deploy: k8s-clean
 	oc start-build build-webconsoleapp --follow
 	oc create -f webconsoleapp-k8s.yaml
 
-.PHONY: containers run clean build k8s-clean k8s-deploy
+patternfly-css:
+	rm -rf $(PATTERNFLY_TMP_DIR)
+	mkdir -p $(PATTERNFLY_TMP_DIR)
+	cd $(PATTERNFLY_TMP_DIR) && npm install @patternfly/patternfly
```

Comment:
This is unversioned -- please pin to a specific version so that this stays reproducible, and can be updated in a controlled fashion.